composer/loggers/mlflow_logger.py

```diff
@@ -139,13 +140,11 @@ def __init__(
         self.run_url = None
 
         if self._enabled:
-            if tracking_uri is None and os.getenv('DATABRICKS_TOKEN') is not None:
-                tracking_uri = 'databricks'
-            if tracking_uri is None:
-                tracking_uri = mlflow.get_tracking_uri()
-            self.tracking_uri = str(tracking_uri)
-            mlflow.set_tracking_uri(self.tracking_uri)
-
+            try:
+                mlflow.login()
```

Comment:
`mlflow.login()` is better for interactive session, like python console, notebook or so. In the composer code I suggest the following for robustness:
```
mlflow.set_tracking_uri("databricks")
```